.teamcity/settings.kts

```diff
@@ -105,6 +105,19 @@ fun Project.buildAll(versionBuild: BuildType) = BuildType {
         }
     }
 
+    features {
+        commitStatusPublisher {
+            vcsRootExtId = "${DslContext.settingsRoot.id}"
+            publisher = github {
+                githubUrl = "https://api.github.com"
+                authType = storedToken {
+                    tokenId =
+                        "tc_token_id:CID_3c2e87590eb212c2b4b597f0b6696f72:34894:ebd4e9b7-fa1d-4e09-b44a-8604fc643a7c"
+                }
+            }
+        }
+    }
+
```

Comment:
It's better not to edit this file directly. It's copied from https://github.com/Kotlin/kotlinx.team.infra/tree/master/main/resources/teamcity periodically, and we'd have to merge changes manually if we don't store project-specific files in `additionalConfiguration`.